### PR TITLE
feat(Client): add stickers method

### PR DESCRIPTION
**Pull request by XEfnexX**
**Add Sticker method (like `<Client>.emojis`) to Client**

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

**Comment by XEfnexX** [comment]
I hope I didn't do anything wrong or forgot something.
It's my first pull requests and I'm open for feedback.

File: src/managers/BaseGuildStickerManager.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const CachedManager = require('./CachedManager');
+const GuildSticker = require('../structures/GuildSticker');
+
+/**
+ * Holds methods to resolve GuildStickers and stores their cache.
+ * @extends {CachedManager}
+ */
+class BaseGuildStickerManager extends CachedManager {
```
[the remark this answers is not in the document]
Comment (SpaceEEC) [reply src/managers/BaseGuildStickerManager.js:10]: Maybe also rename it to `BaseStickerManager` and have `StickerManager` extend it.

**Review by SpaceEEC — CHANGES_REQUESTED** [review]
The new sticker classes aren't adding anything (except `deletable`, which could be moved to `Sticker`), so they should be removed.

**Comment by XEfnexX** [comment]
Well, then my first tried commit was a flop...already thought that.
Thank you. I'll close it now.

**Pull request closed without merging**